src/components/SpatialProfiler/SpatialProfilerComponent.tsx

```diff
@@ -413,7 +413,16 @@ export class SpatialProfilerComponent extends React.Component<WidgetProps> {
                     const wcsLabel = cursorInfo?.infoWCS && !this.lineAxis ? `WCS: ${isXCoordinate ? cursorInfo.infoWCS.x : cursorInfo.infoWCS.y}, ` : "";
                     const xLabel = this.lineAxis ? `${this.lineAxis.label}: ${formattedExponential(nearest.point.x, 5)} ${this.lineAxis.unit ?? ""}, ` : `Image: ${nearest.point.x} px, `;
                     const valueLabel = `${nearest.point.y !== undefined ? formattedExponential(nearest.point.y, 5) : ""}`;
-                    profilerInfo.push("Cursor: (" + wcsLabel + xLabel + valueLabel + ")");
+
+                    const smoothedProfilerInfo = this.genSmoothedProfilerInfo(this.plotData?.smoothingValues);
+
+                    if (smoothedProfilerInfo && this.widgetStore.smoothingStore.isOverlayOn) {
+                        profilerInfo.push(`Cursor: (${wcsLabel}${xLabel}${valueLabel}, Smoothed: ${smoothedProfilerInfo})`);
+                    } else if (smoothedProfilerInfo) {
+                        profilerInfo.push(`Cursor: (${smoothedProfilerInfo})`);
```

Comment:
Do we also add `${wcsLabel}${xLabel}` here?

Reply:
Kuo-song said to leave wcsLabel and I have added xLabel to the smoothedProfilerInfo.

Reply:
Oh I see. Maybe we also remove `${wcsLabel}${imageLabel}` on L440 (at least `imageLabel`?)

Reply:
Yes! Sorry about that.